.vscode/tasks.json

```diff
@@ -30,10 +30,7 @@
 			"presentation": {
 				"panel": "dedicated",
 				"reveal": "never"
-			},
```

Comment:
Since we use esbuild building client code, `$tsc-watch` problemMatcher will not work.